### PR TITLE
Mobile Question Creation Screen

### LINKED ISSUE #505: [Mobile] Implement Question Creation Screen Layout
# 📌 [Mobile] Implement Question Creation Screen Layout
## 🏡 Component 
- [x] Mobile

## 📚 Description
Create the mobile-optimized layout for question creation with proper keyboard handling and responsive design.

## 📋 Acceptance Criteria
- [x] Character count displays
- [x] Tag selection interface
- [x] Difficulty selector
- [x] Form validation
- [x] Loading states
- [x] Error handling

### 📅 Deadline
- **Estimated hours**: 6 hours

### PR DESCRIPTION
**Pull request by atakanyasar**
## 📋 Proposed Changes

  - Implemented create question page

## Related Issue
Closes #505 

## 🧪 Included Tests
 <br>

###After clicking submit:

### CODE REVIEW

File: mobile/app/question/new.tsx [hunk 1]
```diff
@@ -105,12 +134,106 @@ export default function NewQuestionPage() {
               }}
             />
```
Comment (mmtftr) [inline mobile/app/question/new.tsx:135]: Content should be a large input box. Refer to Gluestack [TextArea](https://gluestack.io/ui/docs/components/textarea)

**Review by mmtftr — CHANGES_REQUESTED** [review]
Content should be a large input box. Refer to Gluestack [TextArea](https://gluestack.io/ui/docs/components/textarea)

**Comment by atakanyasar** [comment]
Updated:

**Comment by mmtftr** [comment]
For tag picking can we use this? https://github.com/renrizzolo/react-native-sectioned-multi-select this seems like a better tag selection UI component

**Review by mmtftr — APPROVED** [review]

**Pull request merged**